bridges/snowbridge/primitives/inbound-queue/src/v2/converter.rs

```diff
@@ -64,7 +68,9 @@ pub struct MessageToXcm<
 		ConvertAssetId,
 		GatewayProxyAddress,
 		EthereumUniversalLocation,
-		GlobalAssetHubLocation,
+		AssetHubFromEthereum,
```

Comment:
nit: Can you add a ticket to document the difference between `AssetHubFromEthereum` and `AssetHubUniversalLocation`, with an example values?

The other generic parameters could also do with more documentation.

Reply:
Sure, https://linear.app/snowfork/issue/SNO-1493.

Reply:
In addition to updating the documentation, I'd suggest refactoring and reorganizing the generics here for clarity. 
Original comment: https://github.com/paritytech/polkadot-sdk/pull/8240#discussion_r2060335073 
and I've created a ticket for tracking: https://linear.app/snowfork/issue/SNO-1498